tests/requests/rds/db_snapshot_tests.rb

```diff
@@ -0,0 +1,73 @@
+Shindo.tests('Fog::Rds[:aws] | db snapshot requests', ['aws']) do
+
+  @snapshot_format = {
+    'AllocatedStorage'     => Integer,
+    'AvailabilityZone'     => Fog::Nullable::String,
+    'Engine'               => String,
+    'EngineVersion'        => String,
+    'InstanceCreateTime'   => Time,
+    'DBInstanceIdentifier' => String,
+    'Iops'                 => Fog::Nullable::Integer,
+    'MasterUsername'       => String,
+    'Port'                 => Fog::Nullable::Integer,
+    'Status'               => String,
+    'StorageType'          => String,
+    'SnapshotType'         => String
+  }
+
+  @snapshots_format = {
+    'requestId'   => String
+  }
+
+  @snapshot_copy_result = {
+    'requestId'   => String,
+    'snapshotId'  => String
+  }
+
+  Fog.mock!
```

Comment:
I am unsure if this is required though i couldnt pass the test cases without it..throws a
` Missing required arguments: aws_access_key_id, aws_secret_access_key (ArgumentError)`